socialmediapolicy.md

```diff
@@ -0,0 +1,155 @@
+
+#### Should (recommended)
+* Employees should not connect or engage with any pupils on the current school roll via Social Media.
+* Employees should only engage with parents, pupils and the public with regard to school matters via official school accounts.
+
+#### May (optional)
+* Employees may connect with parents/guardians of a pupil currently on roll if they are known on a personal level.
+* Employees may engage and reply where possible via the official school accounts to increase engagement with others.
+*	Employees may share the content posted to official school accounts on their own accounts if they wish to.
+
+### Passwords and Security
+
+##### Must (required)
+*	Passwords to school social networking accounts must remain safe and secure at all times.
+*	Passwords must not be shared between employees.
+
+##### Should (recommended)
+*	Social Networks should be managed either through a single person or via a social media management solution in which all users will have their own log in and password.
+* Employees should ensure that content is checked by another member of the team before posting to ensure the content abides by the rules contained within this policy.
+
+#### May (optional)
+* Employees may want to set up two factor authentication on their social media accounts for added security.
+
+### Intellectual Property (IP)
+
+##### Must (required)
+*	All media posted must be owned by the person uploading, or the organisation they work for, before it can be uploaded.
+*	Unless stated otherwise in an employment contract, all media and IP (intellectual property) remains the property of the person who created the media. By posting the content to a school account, permission is granted to the school to use the media under the Creative Commons License.
+
+### Personal Accounts on Social Networks
+
+#### Must (required)
+* Employees must not attempt to impersonate the school through an unofficial or spoof account.
+
+#### Should (recommended)
+*	Employees should always treat their personal account posts as public, even if an account has privacy settings enabled which state otherwise, as posts can always be screenshotted and shared.
+
+#### May (optional)
+* Employees may have personal accounts on any social networks they choose.
+
+### Using Personal Devices in School
+
+#### Must (required)
+*	Any personal devices used to hold content from school must be secured with a password or pin.
+*	Any school content kept on a personal device must only be kept for the minimum amount of time feasible, up to a maximum of a week.
+
+#### Should (recommended)
+* Passwords to personal devices should have their pin/passwords changed every 3 months (minimum).
+
+#### May (optional)
+* Employees may use their own devices to create content for social media.
+
+### Parental Responsibilites
+
+#### Must (required)
+* Parents must treat all interactions with an official school account as public.
```

Comment:
Maybe adjust this to...

Parents must treat all interactions with an official school account as public as multiple employees of the school may have access to the social media account, for sensitive subjects it is recommend you speak to the school directly or email the corresponding person.